download.sh

```diff
@@ -0,0 +1,174 @@
+#! /usr/bin/env bash
+USAGE="Usage: ./download.sh [-hD] [-c <NUM>] <DUMP_DIR>
+
+Download the latest Wikipedia Enterprise HTML dumps.
+
+Arguments:
+    <DUMP_DIR>  An existing directory to store dumps in. Dumps will be grouped
+                into subdirectories by date, and a link 'latest' will point to
+                the latest complete dump subdirectory, if it exists.
+
+Options:
+    -h          Print this help screen
+    -D          Delete all old dump subdirectories if the latest is downloaded
+    -c <NUM>    Number of concurrent downloads to allow. Requires MIRROR to be
```

Comment:
`-c 1`, `-c 2` and no option behave in the same way with wget2 installed.

Reply:
Correct, I'll clarify that.